cms/djangoapps/contentstore/utils.py

```diff
@@ -147,6 +147,26 @@ def get_lms_link_for_about_page(course_key):
     )
 
 
+# pylint: disable=invalid-name
+def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
+    """
+    Returns the url to the certificate web view.
+    """
+    assert isinstance(course_key, CourseKey)
+
+    if settings.LMS_BASE is not None:
```

Comment:
I would change this line to test for "is None" and return None if the condition is met.